loadtest/config.go

```diff
@@ -95,14 +95,31 @@ func GetConfig() (*Config, error) {
 	return cfg, nil
 }
 
-// IsValid checks whether a config is valid or not.
+// IsValid reports whether a config is valid or not.
 func (c *Config) IsValid() (bool, error) {
 	if c.ConnectionConfiguration.ServerURL == "" {
```

Comment:
I am wondering if it makes sense for each Config type to have its own `IsValid()` method.
For example we could have a `func (c ConnectionConfiguration) IsValid(bool, error)` and leverage that here.

Reply:
Done.